debian/control

```diff
@@ -54,11 +54,14 @@ Architecture: any
 Depends: adduser,
          apparmor (>= 2.10.95-0ubuntu2.2),
          ca-certificates,
+         cgroup-lite,
          gnupg1 | gnupg,
          snap-confine (= ${binary:Version}),
          squashfs-tools,
-         systemd,
+         sudo (>=1.8.9p5-1ubuntu1.3),
```

Comment:
This makes me slightly uneasy. We don't really need sudo for snapd feels not right to list it here just to get the right version pulled in. I guess we would need a conditional dependency here: if-installed: sudo (>= ...) or something. Not sure what we can do to fix this. One way would be a control.in that generates different control files depending on the release in the clean target. This would also solve the util-linux nitpick below.